### PR TITLE
Silver Solidification

### PR DESCRIPTION
**Pull request by Risikio**
Adds liquid silver as a chemical, adds in process to solidify, and changes Silver-Bells to take advantage of this.

### CODE REVIEW

**Comment by ThePainkiller** [comment]
I always wondered why we didn't have liquid silver on the chem dispensers then I remembered it's a tg thing
Could use liquid silver for a couple more (medical) chemical recipes in the future to make it more varied in use...good idea! I like it!

**Comment by Risikio** [comment]
> I always wondered why we didn't have liquid silver on the chem dispensers then I remembered it's a tg thing
> Could use liquid silver for a couple more (medical) chemical recipes in the future to make it more varied in use...good idea! I like it!

It was Trilby's idea. I was just going to gut silver bells for game balance reasons.

File: code/modules/reagents/recipes/recipes.dm [hunk 1]
```diff
@@ -398,6 +398,15 @@
 	new /obj/item/stack/material/uranium(get_turf(holder.my_atom), created_volume)
 	return
 
+/datum/chemical_reaction/silversolidification
+	result = null
+	required_reagents = list("iron" =5, "frostoil" =5, "silver" =5)
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/recipes/recipes.dm:403]: ?
```suggestion
	required_reagents = list("iron" =5, "frostoil" =5, MATERIAL_SILVER = 20)
```

File: code/modules/reagents/reagents/dispenser.dm [hunk 2]
```diff
@@ -46,6 +46,15 @@
 	reagent_state = SOLID
 	color = "#A8A8A8"
 
+/datum/reagent/metal/silver
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/dispenser.dm:49]: Why is this on dispenser.dm as if it were a chem dispenser reagent, yet it's not added to the list of chems on the dispenser file? Shouldn't you put it on `code/modules/reagents/reagents/other.dm` ?

Comment (Risikio) [reply code/modules/reagents/reagents/dispenser.dm]: So upon further examination... liquid silver was already defined in reagents/other.dm so there wasn't any need to actually redefine it in dispenser.dm... Whoops

**Pull request merged**